src/NCronJob/Registry/JobDefinition.cs

```diff
@@ -2,14 +2,15 @@
 
 namespace NCronJob;
 
-internal sealed record JobDefinition(
+internal record JobDefinition(
```

Comment:
Is there a reason that you unsealed the class, just out of curiosity.

Reply:
I was using it to define the TElement in the `ObservablePriorityQueue` so it couldn't be sealed. However since I merged your changes where you separated out the JobRun from the JobDefinition, I'm now using JobRun so this can be sealed again.